### DISCUSSION TITLE
How about remnote multi-line style?

### DISCUSSION THREAD

**Discussion by zhixiangCai** [st3v3nmw/obsidian-spaced-repetition#192] [General]
The  "?" multi-line style is **not compatible with the Outliner-model(list formating)**, however, for a Outliner user, if the  **remnote multi-line style(:::)**  can be used to create a flashcard with a answer having more than one paragrahs. it would be very exciting. I aslo think there are many people like me are happy the "Spaced Repetion" can have this feature.

Ref of remnote multi-style: [Creatring flashcards](https://www.remnote.io/p/help/document/gdk2ymWq8oHrkXCXb)

**Comment by st3v3nmw** [comment]
Cool, I'll look into this soon.

**Reply by DryIce1** [reply]
Hi @zhixiangCai, thanks for the [Remnote Creating Flashcards link](https://www.remnote.io/p/help/document/gdk2ymWq8oHrkXCXb) - I think their set up is very clear and I would love to adopt this style in Obsidian with this plugin. I also use Outliner Plugin, and want to be able to see the parent lines above the flashcard, and thus look forward to support for #187. I would have stayed with RemNote last year, but I wanted offline markdown files as provided by Obsidian.

Thanks so much for your plugin, @st3v3nmw! I know you have a lot on your roadmap, but do you plan to implemnt the Remnote flashcard types described below? Should this discussion be converted to a feature request? Thanks for considering.

---
- Remnote types of flashcards - reference
	- **Basic Card**  `>>` - forwards only.  
	- **Concept Card** `::`  - forwards and backwards. For style, user should make the concept bold.  
	- **Descriptor Card** `;;` - forwards only. For style, user should make the descriptor in Italics.
	- **Cloze Card** `{{` 
	- **Multiline flashcard** - Turn any card (Basic Card/Concept Card/Descriptor Card) into a card with multiple lines of answer by typing the trigger 3 times 
		- Triggers - `>>>, :::, ;;;`
		- Multiline display all answers at once - use bullet list
		- Multi-line display answers one at a time - use numbered list
- All flashcard types show the ancestor/parent lines above for context, but not siblings.

--- 
**example of RemNote Type flashcards in use**
- History
	- Berlin 
		- what year did the Berlin wall fall? >> 1989
- **Mitochondria** :: Center of energy production in the cell
	- *wall structure* ;; double-membrane-bound organelle
- What are the 20 Rules for Knowledge >>>
	- don't learn things that you do not understand
	- learn before you memorize
	- start from basics before going into complexities
	- keep questions simple
	- pictures assist memory
	- learn mnemonic techniques, e.g. peg lists
	- avoid lists, sets and enumerations (they can be tackled with cloze deletion as in the video)
	- personalize and provide examples
	- cloze deletion is fast and has a great mnemonic power
	- use redundancy (similar items asking similar questions from different angles)
	- use references

**Reply by berot3** [reply]
thanks, RemNote style looks super interesting! Id love to have it :))

**Comment by berot3** [comment]
was looking for exactly something like this!!

just now I structered all my qustions in bullet points. the benefit: i can collapse them and scroll quickly through all questions with the ansers collapsed below the question.

I converted all bulllet points to heading, since they are collapable as well, but had to use blockquote for the answer, since they also included blank lines...

i guess it depends also on how many answer you have for one question :D
